app/assets/javascripts/discourse/app/lib/keyboard-shortcuts.js

```diff
@@ -549,7 +549,15 @@ export default {
         // If effective, prevent default.
         e.preventDefault();
       }
-      $sel.click();
+
+      // If there is more than one match for the selector, just click
+      // the first one, we don't want to click multiple things from one
+      // shortcut.
+      if ($sel.length > 1) {
+        $sel[0].click();
+      } else {
+        $sel.click();
```

Comment:
The logic here doesn't feel right to me, `$sel = $(selector);` should always return a list so I think we can just do `$sel[0].click()` no matter the length of the list.